README.md

```diff
@@ -1,5 +1,40 @@
 # iRODS C++ REST Mid-Tier API
 
+This REST API is designed to be run alongside an iRODS Server to provide an HTTP REST interface to the majority of administrative functions for an iRODS Zone.
+
+It will run under the same linux service account and access `/etc/irods/server_config.json` and the active authenticated `rodsadmin` iRODS account.
```

Comment:
**will** makes the statement sound optional.

The REST API must run under the service account using the authenticated `rodsadmin` account. I feel we need a name that clearly identifies the **first iRODS account** as the **super account**.

Reply:
removed the 'will'

Reply:
it doesn't have to be the first/super irods account... just the active rodsadmin one being used.